Library/Homebrew/keg.rb

```diff
@@ -137,18 +137,19 @@ def to_s
   LIBTOOL_EXTENSIONS = %w[.la .lai].freeze
 
   # Given an array of kegs, this method will try to find some other kegs
-  # that depend on them. If it does, it returns:
+  # or casks that depend on them. If it does, it returns:
   #
   # - some kegs in the passed array that have installed dependents
   # - some installed dependents of those kegs.
   #
   # If it doesn't, it returns nil.
   #
-  # Note that nil will be returned if the only installed dependents
-  # in the passed kegs are other kegs in the array.
+  # Note that nil will be returned if the only installed dependents of the
+  # passed kegs are other kegs in the array or casks present in the casks
+  # parameter.
   #
   # For efficiency, we don't bother trying to get complete data.
-  def self.find_some_installed_dependents(kegs)
+  def self.find_some_installed_dependents(kegs, casks: [])
```

Comment:
I'd rather avoid this method on `Keg` take `casks` because kegs and casks are completely separate concepts. I'd suggest this ensure function move to a new `installed_dependents.rb` or similar.